arch/arm/boot/dts/overlays/revpi-connect4-overlay.dts

```diff
@@ -0,0 +1,294 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/dts-v1/;
+/plugin/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+#include <dt-bindings/pinctrl/bcm2835.h>
+
+/ {
+	compatible = "brcm,bcm2711";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-connect4", "brcm,bcm2711";
+
+			pibridge {
+				pinctrl-names = "default";
+				compatible = "kunbus,pibridge";
+				pinctrl-0 = <&digital_io_pins>;
+
+				/* digital IN, digital OUT */
+				connect-gpios = <&gpio 6 GPIO_ACTIVE_HIGH>,
+						<&gpio 16 GPIO_ACTIVE_HIGH>;
+
+				left-sniff-gpios = <&expander_core 8 GPIO_ACTIVE_HIGH>,
+						   <&expander_core 9 GPIO_ACTIVE_HIGH>;
+				right-sniff-gpios = <&expander_core 11 GPIO_ACTIVE_HIGH>,
+						   <&expander_core 12 GPIO_ACTIVE_HIGH>;
+			};
+
+			leds {
+				compatible = "gpio-leds";
+				power_red {
+					label = "power:1:fault";
+					gpios = <&expander_core 0 GPIO_ACTIVE_HIGH>;
+					linux,default-trigger = "power_red";
+				};
+				a1_red {
+					label = "a1:red:status";
+					gpios = <&expander_core 1 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a1_red";
+				};
+				a1_green {
+					label = "a1:green:status";
+					gpios = <&expander_core 2 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a1_green";
+				};
+				a1_blue {
+					label = "a1:blue:status";
+					gpios = <&expander_core 3 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a1_blue";
+				};
+				a2_red {
+					label = "a2:red:status";
+					gpios = <&expander_core 4 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a2_red";
+				};
+				a2_green {
+					label = "a2:green:status";
+					gpios = <&expander_core 5 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a2_green";
+				};
+				a2_blue {
+					label = "a2:blue:status";
+					gpios = <&expander_core 6 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a2_blue";
+				};
+				a3_red {
+					label = "a3:red:status";
+					gpios = <&expander_power 0 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a3_red";
+				};
+				a3_green {
+					label = "a3:green:status";
+					gpios = <&expander_power 1 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a3_green";
+				};
+				a3_blue {
+					label = "a3:blue:status";
+					gpios = <&expander_power 2 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a3_blue";
+				};
+				a4_red {
+					label = "a4:red:status";
+					gpios = <&expander_power 8 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a3_red";
```

Comment:
a4 and a5 rgb linux,default-trigger is woring (copy'n'paste error)

Reply:
good catch